src/stores/Network.jsx

```diff
@@ -16,7 +17,7 @@ export default class NetworkStore {
   @observable outOfSync = true;
   @observable hw = {active: false, showSelector: false, option: null, derivationPath: null, addresses: [], loading: false, error: null, network: ""};
   @observable downloadClient = false;
-
+  isMobile = checkIsMobile.any;
```

Comment:
Seems to be unused?

Reply:
`isMobile` is used in a couple places:

https://github.com/makerdao/scd-cdp-portal/pull/13/files#diff-ddecf30a0df211dedae25075c43a8decR32

https://github.com/makerdao/scd-cdp-portal/pull/13/files#diff-9942d80cffc84ac661527e3a0ba23903R106

and `network` is available as a `prop` thanks to the `@inject` decorator from mobx